Sources/Errors.swift

```diff
@@ -33,18 +33,3 @@ extension TaskError: CustomStringConvertible {
 		}
 	}
 }
-
-extension TaskError: Equatable {
```

Comment:
is this auto synthesized now?

Reply:
> is this auto synthesized now? from TaskError: Error, Equatable ?

Yes

> I saw no tests for this. Do you think it's necessary to add one?

Nah. This is tested in the compiler.